frontend/src/translations/bg/about.json

```diff
@@ -5,6 +5,10 @@
   },
   "main": {
     "title": "За нас",
+    "betaWarning": "Приложението все още е в бета версия! Ако откриете грешки, не се колебайте да ги докладвате.",
```

Comment:
Is this ‘betaWarning’ translation used? If not, remove it.

Reply:
i refactored with Trans as RyotaMitaraiWeb advised and removed unused translations.